### ISSUE TITLE
loss = nan이 뜨는 경우

### ISSUE THREAD

**Issue by kimwonill** [LISTatSNU/FastMRI_challenge#162]
epoch 2~3부터 중간에 loss = nan이 뜨다가 loss가 무한으로 발산하는데 원인을 잘 모르겠습니다. 이건 무슨 경우인지 알 수 있을까요?

**Comment by ahagyue** [comment]
어떤 코드를 사용하고 계시나요?

**Comment by kimwonill** [comment]
varnet 코드 이용하고 있습니다.

**Comment by ahagyue** [comment]
혹시 코드를 고친 부분은 없나요?
